### PR TITLE
Font performance variant for /firefox/new/

### PR DESCRIPTION
**Pull request by jpetto**
Displays copy while font files are loading. Uses system font for devices less than 760px wide.

@alexgibson r?

### CODE REVIEW

**Comment by jpetto** [comment]
Tested in Fx, Chrome, Safari, IE 10, 11, & Edge. IE 7, 8, 9 VMs all failed to connect to internet. :disappointed:

**Comment by alexgibson** [comment]
@jpetto looks like we have some redirect failures ^ :(

**Comment by alexgibson** [comment]
I managed to test in IE9 and fonts loaded async as expected :+1:

**Comment by alexgibson** [comment]
fontfaceobserver.js seems to throw an `unexpected identifier` error in IE8. Should this perhaps need to be in a separate bundle? Would it be good to test this on a demo server perhaps?

File: media/js/firefox/new-v1.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        if (mq.matches) {
+            var fntOpenSans = new FontFaceObserver('Open Sans', {
+                weight: 'normal'
+            });
+            var fntOpenSansBold = new FontFaceObserver('Open Sans', {
+                weight: 'bold'
+            });
+            var fntOpenSansLight = new FontFaceObserver('Open Sans Light');
+
+            Promise.all([fntOpenSans.check(), fntOpenSansLight.check()]).then(function() {
```
Comment (alexgibson) [inline media/js/firefox/new-v1.js]: Docs suggest we may need to include a `Promise` polyfill for browsers that don't have native support: https://github.com/bramstein/fontfaceobserver. Do we need to do this?

**Comment by alexgibson** [comment]
Seems to work well nice - including the library seems to throw an error in IE < 9, which we probably want to avoid. Also one question about whether we need a Promises polyfill. Aside from that, things look good :+1: 

The test failures can hopefully be fixed too

File: bedrock/firefox/views.py [hunk 2]
```diff
@@ -682,3 +683,18 @@ def get_template_names(self):
...
+@xframe_allow
+def new(request):
+    locale = request.locale
+    variant_locales = ['en-US', 'de']
+
+    # check for variant in querystring
+    variant = request.GET.get('v', '')
+
+    # ensure variant is one of 2 accepted values
+    if (variant not in ['1', '2'] or locale not in variant_locales):
```
Comment (alexgibson) [inline bedrock/firefox/views.py]: Is variant 2 being used?

File: media/js/firefox/new-v1.js (hunk 1 above)
Comment (jpetto) [inline media/js/firefox/new-v1.js]: According to [the docs](https://github.com/bramstein/fontfaceobserver#installation), the version of `fontfaceobserver.js` used includes a small promise polyfill, so should be good there.

File: bedrock/firefox/views.py (hunk 2 above)
Comment (jpetto) [inline bedrock/firefox/views.py]: Variant 2 not being used yet, but likely will be early next week/at next push.

**Comment by alexgibson** [comment]
Tested IE6-8 and page functions as expected. Tested IE9-10, Firefox, Chrome, Safari and font's load async as expected :+1:

**Comment by jpetto** [comment]
Closing in favor of better named #3647.

**Pull request closed without merging**